proect_on_desktop.css

```diff
@@ -0,0 +1,175 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    margin: 0;
+    font-family: Montserrat;
+    background-color: black;
+    color: white;
+}
+
+ul {
+    list-style: none;
+}
+
+.a_little_down_cart {
+    margin-top: 3px;
+}
+
+.container {
+    max-width: 1050px;
+    margin: 0 auto;
+    text-align: center;
+    position: relative;
+}
+
+.hero {
+    background: url('img/fon.jpg') no-repeat;
```

Comment:
don't forget about dark barkdrop on the header background.

Reply:
I don’t really understand what you mean, I’ll also remind you in today’s lecture